### ISSUE TITLE
Long press often doesn't do anything

### ISSUE THREAD

**Issue by Mikaela** [Dax89/harbour-webpirate#48]
I am not sure how to reproduce this, but sometimes when browsing the web, I can try long pressing forever and I never get the long press menu with `Open in new tab` etc. Short pressing still works, but it doesn't help so much as I would like to keep the old tab open.

**Comment by Dax89** [comment]
I have also experienced this bug few times, do you have an example website so I can test it?

**Comment by Mikaela** [comment]
No, it seems to happen totally randomly. Most of time I encounter it annoying way I am on some search engine, mostly [DuckDuckGo](https://duckduckgo.com/) (or if I don't want what I want there), [Google via DDG !g](https://encrypted.google.com/).

**Comment by Dax89** [comment]
Ok!
Thanks for bring this issue there, I have almost missed it! :)

**Comment by rinigus** [comment]
I have this issue at almost all links. For example, when trying to zoom in on links shown above and holding them, nothing happens. While holding, sometimes link flicks into selection with background highlighted, but disappears fast. Short click works. I'm using version 2.0, on nexus 4. How can I help to debug it? Since it happens always, that should help debugging...
